Fedora/notepadqq.spec

```diff
@@ -1,11 +1,11 @@
 Name:			notepadqq
 Version:		1.0.1
-Release:		4%{?dist}
-Summary:		A Linux clone of Notepad++
+Release:		5%{?dist}
+Summary:		A qt advanced text editor
```

Comment:
Better :) I renamed from notepad++ because it seems that not all people know that Notepad++ is gpl